product-scenarios/6-rollout-api-updates-seamlessly/6.1-rollout-api-updates-to-existing-api/6.1.1-update-published-api-using-api-publisher/src/test/java/org/wso2/am/scenario/tests/update/published/api/using/publisher/UpdatePublishedAPITest.java

```diff
@@ -0,0 +1,228 @@
+package org.wso2.am.scenario.tests.update.published.api.using.publisher;
+
+import io.swagger.models.HttpMethod;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONString;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.am.scenario.test.common.APIPublisherRestClient;
+import org.wso2.am.scenario.test.common.APIRequest;
+import org.wso2.am.scenario.test.common.APIStoreRestClient;
+import org.wso2.am.scenario.test.common.ScenarioTestBase;
+import org.wso2.am.scenario.test.common.swagger.*;
+import org.wso2.carbon.apimgt.api.model.APIIdentifier;
+import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.UUID;
+
+public class UpdatePublishedAPITest extends ScenarioTestBase {
+
+    private APIPublisherRestClient apiPublisher;
+    private APIStoreRestClient apiStore;
+    private Properties infraProperties;
+    private List<APIIdentifier> createdAPIs = new ArrayList<>();
+
+    private static final String ADD_RESOURCE = "/add";
+    private static final String RETRIEVE_RESOURCE = "/retrieve";
+    private static final String UPDATE_RESOURCE = "/update";
+
+    private static final String API_KEY = "api";
+    private static final String TEMPLATES_KEY = "templates";
+    private static final String RESOURCES_KEY = "resources";
+
+
+    private static final String ADMIN_USERNAME = "admin";
+    private static final String ADMIN_PASSWORD = "admin";
+
+    private static final String CREATOR_USERNAME = "admin";
+    private static final String CREATOR_PASSWORD = "admin";
+
+    private static final String PUBLISHER_USERNAME = "admin";
+    private static final String PUBLISHER_PASSWORD = "admin";
+
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        infraProperties = getDeploymentProperties();
```

Comment:
Following logic has been moved to parent class constructor. So we can remove it from here